iocore/net/UnixNetVConnection.cc

```diff
@@ -1354,6 +1320,12 @@ UnixNetVConnection::connectUp(EThread *t, int fd)
 void
 UnixNetVConnection::clear()
 {
+  // clear timeout variables
+  vc->next_inactivity_timeout_at = 0;
+  vc->next_activity_timeout_at   = 0;
+  vc->inactivity_timeout_in      = 0;
+  vc->active_timeout_in          = 0;
```

Comment:
What is `vc` here? Are those `this`?

Reply:
Yes. I have rebased this pr, please review. thanks.

Please feel free to read & check the comments at the beginning of NetHandler.